__tests__/RequestVerifier.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { describe, expect, test, beforeAll } from "@jest/globals";
+import * as moment from 'moment';
+import { RequestVerifier } from '../utils/RequestVerifier'
+
+describe("RequestVerifier", () => {
+    let instance: RequestVerifier
+    beforeAll(() => {
+        instance = new RequestVerifier();
+    });
+    describe("#checkRequestBody", () => {
+        describe("Request Body is empty or null", () => {
```

Comment:
break out into multiple tests

Reply:
I do not think we need to do a null test due to the fact that typescript has strictNullChecks and since we have typed requestBody to string, null can't be passed in. Therefore there might not be a possibility that null will get passed in and we may not need a null test case.

https://www.typescriptlang.org/tsconfig#strictNullChecks